packages/open-truss/bin/bin/hello_world

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Should these files use kebab casing? IIRC that matches the guidelines for this project. It's also much more standard for cli/binary commands in my experience.

Reply:
- `hello_world -> hello-world`
- `tsc_watch -> tsc-watch`